docs/source/models/concept_models/vivarium_moud/concept_model.rst

```diff
@@ -131,23 +138,124 @@ Key features of the model include:
+     - Name
+     - Definition
+   * - pr_uh
+     - Rate of becoming unhoused
+     - Rate at which individuals transition from private residence to being unhoused.
+   * - pr_inc
+     - Rate of incarceration (from residence)
+     - Rate at which individuals transition from private residence to incarceration.
+   * - uh_pr
+     - Rate of obtaining housing
+     - Rate at which unhoused individuals transition to private residence.
+   * - uh_inc
+     - Rate of incarceration (from unhoused)
+     - Rate at which unhoused individuals transition to incarceration.
+   * - inc_pr
+     - Rate of release to housing
+     - Rate at which incarcerated individuals are released to private residence.
+   * - inc_uh
+     - Rate of release to unhoused
+     - Rate at which incarcerated individuals are released into unhoused status.
+
+Key features and interactions of this submodel include:
+
+- **Interdependence with OUD Status:** Transition rates *within* this Quarters Model (e.g., `pr_uh`, `uh_inc`) may be influenced by the individual's state in the Core Disease Model (susceptible, with condition, on treatment). For example, individuals with untreated OUD might have a higher rate of becoming unhoused.  See below for details on how this is implemented using the :code:`RiskEffect` component.
+- **Influence on OUD Transitions:** Conversely, an individual's state in the Quarters Model can affect transition rates *within* the Core Disease Model. For instance, being unhoused or incarcerated might increase OUD incidence risk (`i`), decrease treatment initiation rates (`ti`), or increase treatment failure rates (`tf`). See below for details on how this is implemented using the :code:`RiskEffect` component.
+- **Parameterization:** Rates need to be parameterized using available data sources on housing instability, homelessness, incarceration, and release patterns, potentially stratified by relevant demographic factors and OUD status where data permits.
+
 
 4 Data Notes
 ++++++++++++
 
-The model requires the following data streams:
+Parameterizing the MOUD simulation requires integrating data from various sources to define initial population states and the transition rates governing movement between states in both the Core Disease Model and the Quarters Model. Key data requirements and methodologies include:
+
+4.1 Core Disease Model Parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Core Disease Model requires estimates for:
+
+* **Overall OUD Prevalence:** Sourced from the most recent Global Burden of Disease (GBD) Study, specifying the relevant year and location.
+* **Treatment Coverage Ratios:** Data on the proportion of individuals with OUD who are receiving MOUD. Sources may include national surveys (e.g., NSDUH), jail healthcare system data, and HIV surveillance studies.
+* **Transition Rates:** Estimates for disease incidence (`i`), untreated remission (`r`), treatment initiation (`ti`), treatment failure (`tf`), and treatment success (`ts`).
```

Comment:
I have a feeling these backticks were supposed to be for code-formatting/monospace, which needs double-backticks in RST.

Reply:
good catch, thanks